front/app/containers/Admin/formBuilder/components/FormFields/index.tsx

```diff
@@ -67,72 +63,167 @@ const FormFields = ({
     watch,
     formState: { errors },
   } = useFormContext();
-  const formCustomFields: IFlatCustomField[] = watch('customFields');
   const locale = useLocale();
+  const formCustomFields: IFlatCustomField[] = watch('customFields');
+
   if (isNilOrError(locale)) {
     return null;
   }
 
+  const FormFieldsContainer = styled(Box)`
+    &:hover {
+      cursor: pointer;
+    }
+  `;
+
+  const isFieldSelected = (fieldId) => {
```

Comment:
Wondering if I should move the functions declared between lines 78 - 104 to another util file maybe? Feels a bit messy like this?

Reply:
Yes, If we can move them then that would be ideal. I'd also add types to all of them.